PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.java

```diff
@@ -56,10 +56,19 @@ void sendRequest(final Context context, final PayPalRequest payPalRequest,
                         ? SETUP_BILLING_AGREEMENT_ENDPOINT : CREATE_SINGLE_PAYMENT_ENDPOINT;
                 String url = String.format("/v1/%s", endpoint);
 
-                String requestBody =
-                        payPalRequest.createRequestBody(configuration,
-                                braintreeClient.getAuthorization(),
-                                successUrl, cancelUrl);
+                String universalLink = null;
```

Comment:
PR #1085 makes app links required for all PayPal flows in v5, so I think we can simplify this logic and just pull the `appLinkReturnUrl` for all cases. Let's also stick with `appLink` naming since universal links are iOS specific

Reply:
Or even more simplified, let's set this parameter along with `cancelUrl and `successUrl` within the constructor

Reply:
Updated, let me know what do you think, thanks!!! [945586d](https://github.com/braintree/braintree_android/pull/1086/commits/945586dfc862319dc57bbd2050d544b9fedbac2d)